product_attribute_company_favorite/models/product_attribute.py

```diff
@@ -0,0 +1,78 @@
+from odoo import api, fields, models
+
+
+class ProductAttributeIsFavoriteMixin(models.AbstractModel):
```

Comment:
please, split the file into 4 files. (one per class).

Reply:
done